recipes-core/packagegroups/packagegroup-rpi-test.bb

```diff
@@ -17,13 +17,16 @@ RDEPENDS_${PN} = "\
     rpi-gpio \
     pi-blaster \
     python3-rtimu \
-    python3-sense-hat \
     connman \
     connman-client \
     wireless-regdb-static \
     bluez5 \
 "
 
+RDEPENDS_${PN} += "\
+    ${@bb.utils.contains("BBFILE_COLLECTIONS", "meta-python", "python3-sense-hat", "", d)} \
+"
```

Comment:
Could this be done via a `packagegroup-rpi-test.bbappend` file under `dynamic-layers/meta-python/...` to keep things grouped together?

Reply:
I would also like that. It feels like we just break that package when meta-python is not available.

Reply:
Yes, I thought of that but this seems more directional when someone wanted to use this package that they would need meta-python, shoving all into dynamic-layers would make it more cryptic.

Reply:
moved now